manifests/no-script-v2/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "NoScript",
+  "url": "https://secure.informaction.com/download/releases/noscript-11.4.44-chrome.zip",
```

Comment:
How do we plan on keeping this one up-to-date with the latest version from https://noscript.net/getit/#nonstore-versions-for-chromium? Updating manually on some schedule?

Reply:
Manually until they provide us the `latest` link